### ISSUE TITLE
Not Launching Game [ Clicking Play does update again ]

### ISSUE THREAD

**Issue by Smugger21** [SkyrimTogether/issues-launcher#51]
## Description

It was working (loading skyrim) when I first got it, but now, it does the integrity check, downloads the update, then starts seeding, I click play and then it says launching, then it starts downloading the exact same update that is already had (I check the files and the versions and size it just changes the last modified date). Then is starts seeding again. Then I try play again and the process repeats. 

## Steps to reproduce

How to reproduce this issue:

1. Start the launcher.
2. Wait for update
3. Click Play - Repeats 

## Reproduction rate

How many times can you reproduce it?
It does it every time right now

## Expected result

Launch Skyrim

## Your environment

* Launcher version:
1.0.6

* Game edition (choose on which edition do you have problems):
    * The Elder Scrolls V: Skyrim

* Operating system (choose one):
    * Windows 10 x64

## Evidence (optional)

Let me know if you need a video, I will post it

## Final checks for the reporter

- [x] I have browsed through the other issues and confirmed that this issue has not been reported already (There is a semi-similar one here but no updates on it and I think that issue was fixed) Here is the link: https://github.com/SkyrimTogether/issues-launcher/issues/35
- [x] I made sure that I have applied the template for my bug report

**Comment by Smugger21** [comment]
CRUD, there is this one this pretty close also but they don't say anything about getting the play button then is says launching then downloads the update again. https://github.com/SkyrimTogether/issues-launcher/issues/16

**Comment by Smugger21** [comment]
I found that the issue in my case was the AV removing the st_loader file.
I had already unblocked the Harbor file from the AV when I first installed the Launcher, and the game would launch from the launcher but not sure what in the mod has change recently that would make the AV think it is a virus. 

But ya, that fixed my issue, I am closing this issue, you might want to check with the other people to see if that is the same for them.

So TL;DR
- Check you Antivirus program virus vault and see if st_loader or any other file from the mod is in there and restore it and add an exception for it.
